hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemFlush.java

```diff
@@ -208,43 +208,44 @@ public Void call() throws Exception {
   }
 
   @Test
-  public void testFlushWithFlushEnabled() throws Exception {
-    testFlush(true);
+  public void testFlushWithOutputStreamFlushEnabled() throws Exception {
```

Comment:
How about adding tests for combinations of `supportFlush `and `disableOutputStreamFlush` ?

Reply:
As the configs are kept separate for HFlush + Hsync and Flush, new tests focus only on the new OutputStream Flush config and the Flush API calls. Will make an update based on the decision on above comment.